src/dynamic/index.tsx

```diff
@@ -0,0 +1,51 @@
+import loadJS from "load-js";
+import API from "../api";
+import Flipside from "..";
+import template from "lodash/template";
+import mapValues from "lodash/mapValues";
+
+type DynamicOpts = {
+  widgetId: string;
+  assetId: string;
+};
+
+export default async function dynamic(api: API, el: string, opts: DynamicOpts) {
+  const res = await api.fetchDynamic(opts.widgetId);
+  if (!res.success) {
+    throw new Error(`Flipside: dynamic widget loading failed`);
+  }
+
+  await loadJS([
+    {
+      url: "https://d3sek7b10w79kp.cloudfront.net/flipside-v1.10.0.js",
```

Comment:
I don't think this should be hard-coded. The library to load should come from the API.

Reply:
Whoops, good catch.